### DISCUSSION TITLE
crash gta 5 translation file

### DISCUSSION THREAD

**Discussion by SpinDex** [MAFINS/MenyooSP#668] [Q&A]
Hello, my name is Spin and I'm new here. I've known about this mod for a long time, but a few days ago I discovered a way to create my own translation for words from English to Portuguese, but when I tried to create the code, it gave an error. I had created the code using chatGpt, and it said that it gave the error because my GTA 5 was out of date or the mod was out of date, but I believe it is not.

Please help me to solve this problem
![Captura de Tela (26)](https://github.com/MAFINS/MenyooSP/assets/166856712/041bb9ef-9556-4c54-b575-84a95afcb1a9)
[Portuguese.json](https://github.com/MAFINS/MenyooSP/files/14965300/Portuguese.json)

**Comment by Marexter** [comment]
try re-downloadiing

**Comment by Dwight258** [comment]
The error seems to occur when ScriptHookVDotNet is included.
Please remove ScriptHookVDotNet and try again.

**Reply by Dwight258** [reply]
If this does not solve the problem, the cause lies elsewhere.

**Comment by Marexter** [comment]
re install the mod
